packages/common/src/Ethers.js

```diff
@@ -0,0 +1,20 @@
+const { hexlify, toUtf8Bytes } = require("ethers/utils");
+
+function stringToBytes32(text) {
```

Comment:
Just curious, is this copied from somewhere? I feel like I recognize the code

Reply:
Got it from this ethers [issue](https://github.com/ethers-io/ethers.js/issues/207#issuecomment-398691570)

Reply:
Ah yep! I think @Tulun used this some snippet in the voter dapp. May be useful (later) to pull this out into a separate lib.

Reply:
Yeah thats why I started this Ethers.js file to start collecting common ethers utils